### ISSUE TITLE
TouchEvent handling seems broken when there is jQuery in the page

### ISSUE THREAD

**Issue by HadrienPierart** [AllanBishop/angular-img-cropper#37]
The line https://github.com/AllanBishop/angular-img-cropper/blob/master/src/angular-img-cropper.js#L1089 seems to break when jquery is in the page because the TouchEvent is wrapped in a jquery event, thus making it only available through the originalEvent property of the event.
Is this something you changed recently ? Im pretty sure that this worked on mobile a few months back but now, its broken on all platforms when jquery is present...

**Comment by gvinaccia** [comment]
pull request #32 that should fix the issue

**Comment by HadrienPierart** [comment]
Yep, that's exactly the fix I did on my fork to settle the issue. Good for me !
